### PR TITLE
fix staticcheck for kubectl pkg files

### PR DESCRIPTION
**Pull request by SaiHarshaK**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fixes pkg/kubectl static checks for
- staging/src/k8s.io/kubectl/pkg/cmd/get/customcolumn.go
- staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go
- staging/src/k8s.io/kubectl/pkg/cmd/testing/fake.go

**Which issue(s) this PR fixes**:
Part of #92402

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by SaiHarshaK** [comment]
The size of the PR is very small but only these files are there in kubectl pkg which are failing static check. If required I can maybe try combining with another package in this PR.

File: staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```
Comment (liggitt) [inline staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go:212]: this actually seems like a bug... I'm not sure whether we should be ignoring this error or not... can you split this into its own PR and tag @seans3 / @soltysh for a review on whether we should handle the error better or ignore it?

**Review by liggitt — COMMENTED** [review]
largely lgtm, one question about ignoring the error

File: staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go (hunk 1 above)
Comment (SaiHarshaK) [reply staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go:212]: Okay, will do! On side note, i think that PR will only have 1 addition then, is it fine?
